components/portfolio/BalanceRow.vue

```diff
@@ -64,7 +64,10 @@
       </div>
     </td>
 
-    <LazySendModal ref="SendModal" :denoms="[balance.denom]" />
+    <LazySendModal
```

Comment:
in theory this element should go to the parent component and be triggered by an event. no need to have x send modal elements in there

Reply:
what do you mean?

Reply:
every balance row imports a lazysendmodal. why? there is only one open at a time.
therefor it would be better to move the sendmodal to the balances component and handle a click on a send button to trigger an event that bubbles to the balances component and opens the modal. but for another PR I guess. it becomes more obvious as you need to pass the balances from the parent now to this child component

Reply:
Yes, it makes much more sense to do it like that

Reply:
@Bitcoinera open a new issue for this refactor

Reply:
oki :+1: